webpack.playground.config.js

```diff
@@ -9,6 +10,14 @@ const PLAYGROUND = path.resolve(__dirname, 'playground');
 const DEFAULT_SERVER_PORT = 3000;
 const WIDGET_RC_FILE = '.widgetrc.js';
 
+if (!fs.existsSync(WIDGET_RC_FILE) && fs.existsSync('.widgetrc')) {
```

Comment:
nit: Since we reuse `!fs.existsSync(WIDGET_RC_FILE)` below, consider bringing this to a variable